README.md

```diff
@@ -32,7 +32,9 @@ It uses the same internal APIs as `adb backup` which is deprecated and thus need
 * `android.permission.INSTALL_PACKAGES` to re-install apps when restoring from backup.
 
 ## Contributing
-Bug reports and pull requests are welcome on GitHub at https://github.com/stevesoltys/seedvault. 
+Bug reports and pull requests are welcome on GitHub at https://github.com/stevesoltys/seedvault.
+
+This project aims to adhere to the [official Kotlin coding style](https://developer.android.com/kotlin/style-guide).
```

Comment:
i.e. the previous question was more is there a reference codestyle file for this?

Reply:
Well, this here is the docs for Android which ktlint checks. As always there's some slight disagreements/bugs between the IDE and ktlint, so some smaller fixes are needed here and there. I am not aware of a reference xml Intellj file for the coding style. The most important bit there is

```xml
<option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
```